added manpage from debian package

junghans opened the pull request:
Closes #1072

rbberger commented on doc/lammps.1:
```diff
@@ -0,0 +1,41 @@
+.TH LAMMPS "2012-02-23"
```
probably should update this date and use the release date?

rbberger commented on doc/lammps.1:
```diff
@@ -0,0 +1,41 @@
+See http://lammps.sandia.gov/ for documentation.
+
+.SH COPYRIGHT 
+© 2003--2012 Sandia Corporation
```
same here 2018

rbberger commented:
@junghans should we add an install to cmake to put it in the right place?

junghans commented:
@rbberger: install rule added

rbberger commented:
great! looks good

junghans commented:
@akohlmey good point, I changed the installation name of the manpage. I also changed to `-in` syntax as suggested.

However, what options should actually go into the manpage, I will leave to @sjplimp.

sjplimp commented:
@junghans all the command-line options are on this page:  doc/src/Run_options.txt.  You can use
whatever info there you like for this man page

rbberger commented:
rerunning checks, was switching to an additional build server but that went wrong...
